src/utils/motebehovUtils.ts

```diff
@@ -179,6 +185,60 @@ export const toMotebehovTilbakemeldingDTO = (
   };
 };
 
-export function isSykmeldtSvar(motebehov) {
+export function isArbeidstakerMotebehov(motebehov: MotebehovVeilederDTO) {
   return motebehov.opprettetAv === motebehov.aktorId;
 }
+
+export function mapMotebehovToMeldtMotebehovFormat(
+  motebehov: MotebehovVeilederDTO[]
+): MeldtMotebehov[] {
+  return motebehov
+    .filter(
+      (motebehov) => motebehov.skjemaType === MotebehovSkjemaType.MELD_BEHOV
+    )
+    .map((motebehov) => {
+      return {
+        id: motebehov.id,
+        opprettetDato: motebehov.opprettetDato,
+        opprettetAv: motebehov.opprettetAv,
+        opprettetAvNavn: motebehov.opprettetAvNavn,
+        innmelder: isArbeidstakerMotebehov(motebehov)
+          ? MotebehovInnmelder.ARBEIDSTAKER
+          : MotebehovInnmelder.ARBEIDSGIVER,
```

Comment:
I stedet for å måtte regne ut dette rundt omkring i løsningen, la jeg på en `innmelder`.